types/jquery/index.d.ts

```diff
@@ -2155,17 +2160,19 @@ interface JQuery<TElement extends Node = HTMLElement> {
      * @see {@link https://api.jquery.com/toggle/}
      * @since 1.0
      */
-    toggle(duration: JQuery.Duration, complete?: (this: TElement) => void): this;
+    toggle(duration: JQuery.Duration, complete: (this: TElement) => void): this;
     /**
      * Display or hide the matched elements.
      *
-     * @param options A map of additional options to pass to the method.
-     *                Use true to show the element or false to hide it.
+     * @param duration_complete_options_display A string or number determining how long the animation will run.
+     *                                          A function to call once the animation is complete, called once per matched element.
+     *                                          A map of additional options to pass to the method.
+     *                                          Use true to show the element or false to hide it.
      * @see {@link https://api.jquery.com/toggle/}
      * @since 1.0
      * @since 1.3
      */
-    toggle(options?: JQuery.EffectsOptions<TElement> | boolean): this;
+    toggle(duration_complete_options_display?: JQuery.Duration | ((this: TElement) => void) | JQuery.EffectsOptions<TElement> | boolean): this;
```

Comment:
I broke with naming convention because I wanted to indicate that they were multiple dissimilar parameters. `durationCompleteOptionsDisplay` might give off the impression that it was some mega-parameter or something.